addons/pandora/backend/entity_backend.gd

```diff
@@ -165,37 +180,56 @@ func get_all_entities(parent:PandoraCategory = null, sort:Callable = func(a,b):
 ## Initialize this backend with the given data dictionary.
 ## The data needs to come from a source that was produced via
 ## the save_data() method.
-func load_data(data:Dictionary) -> void:
+func load_data(data:Dictionary) -> LoadState:
+	_load_state = LoadState.LOADING
 	_root_categories.clear()
 	_categories = _deserialize_categories(data["_categories"])
+	if _categories == null:
+		_load_state = LoadState.LOAD_ERROR
+	if _load_state == LoadState.LOAD_ERROR:
+		return _load_state
 	_entities = _deserialize_entities(data["_entities"])
+	if _entities == null:
+		_load_state = LoadState.LOAD_ERROR
+	if _load_state == LoadState.LOAD_ERROR:
+		return _load_state
 	_properties = _deserialize_properties(data["_properties"])
+	if _properties == null:
+		_load_state = LoadState.LOAD_ERROR
+	if _load_state == LoadState.LOAD_ERROR:
+		return _load_state
 	for key in _categories:
 		var category = _categories[key] as PandoraCategory
 		if category._category_id:
 			if not _categories.has(category._category_id):
 				push_error("Pandora error: category " + category._category_id + " on category does not exist")
-				continue
```

Comment:
Let's not continue - this should **never** happen and instead we need to throw an error.